libpince/utils.py

```diff
@@ -1189,3 +1181,9 @@ def wrapper(*args, **kwargs):
             traceback.print_exc()
 
     return wrapper
+
+
+#:tag:Utilities
+def caps_hex(hex_str: str):
```

Comment:
I would personally change this to `upper_hex`

Reply:
Seconding @brkzlr but also there's another small issue with upper case. Changing the upper case bytes to their original value will trigger a rewrite because of the case mismatch. Try writing 8a to a cell that reads 8A for instance. This check should be case-insensitive

Reply:
changed in upcoming commit